rest_framework/management/commands/generateschema.py

```diff
@@ -6,6 +6,7 @@
 from rest_framework.schemas.openapi import SchemaGenerator
 
 OPENAPI_MODE = 'openapi'
+OPENAPI_AGID_MODE = 'openapi-agid' # WiP
```

Comment:
This is a Work in Progress, do not waste your brain with these changes in management commands